core/templates/components/shared-component.module.ts

```diff
@@ -460,3 +460,5 @@ import { HybridRouterModuleProvider } from 'hybrid-router-module-provider';
 })
 
 export class SharedComponentsModule { }
+
```

Comment:
Drop extra newlines?

Reply:
Forgot to do this one. Will push it soon.

Reply:
Done!